src/wp-includes/option.php

```diff
@@ -643,50 +643,143 @@ function wp_load_alloptions( $force_cache = false ) {
 }
 
 /**
- * Loads and primes caches of certain often requested network options if is_multisite().
+ * Primes specific network options into the cache with a single database query.
```

Comment:
```suggestion
 * Primes specific network options for the current network into the cache with a single database query.
```

Similar to the difference between `get_site_option` and `get_network_option`.